base.py

```diff
@@ -44,7 +45,7 @@ def time_of_day(self):
 
 class City(object):
 
-    def __init__(self, env, n_people, rng, x_range, y_range, start_time, init_percent_sick, Human, sim_days):
```

Comment:
It's not the business of the City to duplicately know how long to run itself (any any units, whether days, seconds, ticks, ...)

The Env (already passed in as first argument) ought to know though.